Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -110,6 +110,18 @@ private void OnExamine(Entity<PlantHolderComponent> entity, ref ExaminedEvent ar
                                 ? "plant-holder-component-plant-old-adjective"
                                 : "plant-holder-component-plant-unhealthy-adjective"))));
                 }
+
+                if (component.Seed.Ligneous)
+                    args.PushMarkup(Loc.GetString("mutation-plant-ligneous"));
+
+                if (component.Seed.TurnIntoKudzu)
+                    args.PushMarkup(Loc.GetString("mutation-plant-kudzu"));
+
+                if (component.Seed.CanScream)
+                    args.PushMarkup(Loc.GetString("mutation-plant-scream"));
+
+                if (component.Seed.Viable == false)
+                    args.PushMarkup(Loc.GetString("mutation-plant-unviable"));
```

Comment:
why are those things hardcoded?

Reply:
The values on the seed? They've always been like that.

The description localization keys? That's how they're used here. There's 15 other 'hard-coded' localization keys in this function. I don't have anywhere else to attach that value to, so this is basically the only way to localize those descriptions.

Reply:
It's about that long if else thing. You should add something like visibletext to (i think) https://github.com/space-wizards/space-station-14/blob/master/Resources%2FPrototypes%2FHydroponics%2FrandomMutations.yml
Then add something like
For (mutation in mutation)
  If mutation.visibletext not empty
    Addtext(getLocale(mutation.visibletext))

Reply:
Oh, that's the thing I did for produce.

That doesn't work for plants right now, because the mutations that change properties on the seed don't stay on a list of persistent mutations. They flip the boolean and aren't kept around because they don't add components or need to be executed again on produce when the plant is harvested.

To make those work like that, I'd need the 4 booleans in question to be their own component/system instead of part of SeedData. This is "Part 2" of my upgrade plan and I'm currently working on that, though its going slow. Screaming is a better candidate to become an effect, but that's enough work that it should be a separate PR instead of jammed in here.